code/game/objects/structures/plasticflaps.dm

```diff
@@ -81,3 +81,7 @@
 
 /obj/structure/plasticflaps/airtight // airtight defaults to on 
 	airtight = 1
+
+/obj/structure/plasticflaps/airtight/Initialize()
+	. = ..()
+	become_airtight()
```

Comment:
```
if(airtight)
	become_airtight()()
```

Reply:
You want that for regular plastic flaps? Only, the airtight child should always be airtight no?

Reply:
Ah, this should indeed instead be
```
/obj/structure/plasticflaps/Initialize()
	. = ..()
	become_airtight() 
```

Reply:
Actually on the subject, should the destroy be checking for airtightness? I don't know what other things cause a turf to block air.